mm2src/mm2_metrics/src/mm_metrics.rs

```diff
@@ -535,4 +521,35 @@ mod test {
 
         block_on(async { Timer::sleep(6.).await });
     }
+
+    //    #[test]
+    //    fn test_prometheus_format() {
```

Comment:
What do you think about uncommenting out the test and marking it as `#[ignored]`?
Just print the prometheus string to the console.